### PR TITLE
[DFW-2024] - Initial setup for 2024 conference

### PR DESCRIPTION
**Pull request by baldwinjim**
Initial setup for 2024 conference

### CODE REVIEW

File: content/events/2024-dallas/propose.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+### If you're not planning to propose a talk, please consider <a href="mailto:dallas@devopsdays.org?subject=Request%20to%20help%20comment%20and%20vote%20on%20CFP%20selections...&body=">asking for permission to comment and cast your vote on proposals submitted by others.</a>
+
+Our main criteria to make it to the top selection are:
+
+- _original content_: content not yet presented at other conferences, or a new angle to an existing problem
+- _new presenters_: people who are new to the space and have insightful stuff to say; we want to hear everybody's voice
+- _no vendor pitches_: as much as we value vendors and sponsors, we just don't think this is the right forum. You can demo at your table or during Open Space.
+
+<strong>How to submit a proposal:</strong> Go to the following website and follow the instructions on that site.<br> 
+## <a href="https://www.papercall.io/dod-dfw-2024">devopsdays DFW 2024 - PaperCall.io</a><br>
```
Comment (phrawzty) [inline content/events/2024-dallas/propose.md:25]: This URL leads to a 404

File: data/events/2024-dallas.yml [hunk 2]
```diff
@@ -0,0 +1,487 @@
...
+
+startdate: 2024-09-04T00:00:00-05:00 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate: 2024-09-05T23:59:59-05:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
+
+# Leave CFP dates blank if you don't know yet, or set all three at once.
+cfp_date_start: # start accepting talk proposals.
+cfp_date_end: # close your call for proposals.
+cfp_date_announce: # inform proposers of status
+
+cfp_link: "https://www.papercall.io/dod-dfw-2024" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
```
Comment (phrawzty) [inline data/events/2024-dallas.yml:21]: This leads to a 404.

File: content/events/2024-dallas/welcome.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+  </div>
+</div> -->
+<div class = "row">
+  <div class = "col-md-2">
+    <strong>Propose</strong>
+  </div>
+  <div class = "col-md-8">
+    {{< event_link page="propose" text="Propose a talk!" >}}
+  </div>
+</div>
```
Comment (phrawzty) [inline content/events/2024-dallas/welcome.md:67]: You probably want to leave this commented until you get your CFP form sorted.

File: data/events/2024-dallas.yml [hunk 4]
```diff
@@ -0,0 +1,487 @@
...
+    label: Media
+    max: 5
+  - id: npo
+    label: Non-Profit
+    max: 10
+  - id: community
+    label: Community
+    max: 10
+
+program:
```
Comment (phrawzty) [inline data/events/2024-dallas.yml:291]: This is… last year's programme, no?

**Review by phrawzty — COMMENTED** [review]
Hello,

This _feels_ like you copied the 2023 event and modified it, which is not what we'd recommend doing, as it introduces too many opportunities for erroneous information and/or unexpected behaviour. In general, it's best to use the `add_new_event.sh` script to add a new event (see [the docs for more info](https://github.com/devopsdays/devopsdays-web/blob/main/utilities/README.md#events)).

My strong recommendation would be to close this PR and use the script to create a new event cleanly, then open a new PR. If you need any help or guidance, we're more than happy to help; feel free to email or reach out on the organiser Slack. 😄

**Comment by baldwinjim** [comment]
@phrawzty I did use the script, but for the sake of not recreating many of the same page layouts and information that we use every year I did copy a lot of the content and files from last year to this year.  I did this exact process last year and had no issues.  Please accept this PR and merge as it takes a lot of later work and gets it done for now.  I understand that there are some URLs that lead to a 404, but those pages that would display those URLs are not enabled as part of this PR.  I am going to update the welcome.md and comment out those links leading to propose and sponsor.  

Please give this a look now after my last commit.

**Pull request merged**